qark/modules/unpackAPK.py

```diff
@@ -56,11 +56,14 @@ def unpack():
             zf.extractall(dirname + "/", zf.namelist(), )
             logger.info('Extracted APK to %s', dirname + '/')
             common.pathToDEX = dirname + "/classes.dex"
+            if not os.path.exists(common.pathToDEX):    # Some default/system APKs from Google don't have this file
+                logger.error("The classes.dex file was not found for this APK! Please select a different APK.")
+                raise Exception
```

Comment:
Ensure that there was a valid classes.dex file found when unpacking the APK - Without this, the dex2jar process fails and the program crashes.